tapalcatl_server/handler.go

```diff
@@ -468,7 +471,14 @@ func MetatileHandler(p Parser, metatileSize int, mimeMap map[string]string, stor
 		reqState.Coord = &parseResult.Coord
 		reqState.HttpData = &parseResult.HttpData
 
-		metaCoord, offset := parseResult.Coord.MetaAndOffset(metatileSize)
+		metaCoord, offset, err := parseResult.Coord.MetaAndOffset(metatileSize, tileSize)
+		if err != nil {
+			configErrors.Add(1)
+			logger.Warning(LogCategory_ConfigError, "MetaAndOffset could not be calculated: %s", err.Error())
+			http.Error(rw, err.Error(), http.StatusInternalServerError)
+			reqState.ResponseState = ResponseState_Error
+			return
```

Comment:
It looks like if it's left as nil, then an error gets logged with "Invalid fetch state". Should I just add a new `FetchState_NotFetched`? Or exempt `FetchState_Nil` from the invalid state check?